MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -959,17 +959,31 @@ private async Task<string> ExtractImageInternal(string inputPath, string contain
                             // We don't actually expect the process to be finished in one timeout span, just that one image has been generated.
                         }
 
+                        try
+                        {
                         cancellationToken.ThrowIfCancellationRequested();
 
                         var jpegCount = _fileSystem.GetFilePaths(targetDirectory).Count();
 
                         isResponsive = jpegCount > lastCount;
                         lastCount = jpegCount;
+                        }
+                        catch (OperationCanceledException)
```

Comment:
Instead of doing this, can we just check `cancellationToken.IsCancellationRequested`, instead of throwing and catching in this loop?

Reply:
That should work too, I'll test it now.